ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListSourceExecutor.java

```diff
@@ -241,27 +262,52 @@ private ListSourceExecutor() {
         .collect(Collectors.toList());
   }
 
+  private static SourceDescriptionWithWarnings describeSource(
+      final KsqlConfig config,
+      final KsqlExecutionContext executionContext,
+      final ServiceContext serviceContext,
+      final SourceName table,
+      final boolean extended,
+      final ConfiguredStatement<ShowColumns> statement,
+      final SessionProperties sessionProperties
+  ) {
+    return describeSource(
+        config,
+        executionContext,
+        serviceContext,
+        table,
+        extended,
+        statement,
+        sessionProperties,
+        ImmutableList.of()
+    );
+  }
+
   private static SourceDescriptionWithWarnings describeSource(
       final KsqlConfig ksqlConfig,
       final KsqlExecutionContext ksqlEngine,
       final ServiceContext serviceContext,
       final SourceName name,
       final boolean extended,
-      final String statementText) {
+      final ConfiguredStatement<? extends StatementWithExtendedClause> statement,
+      final SessionProperties sessionProperties,
+      final Collection<RemoteSourceDescription> remoteSourceDescriptions
+
+  ) {
     final DataSource dataSource = ksqlEngine.getMetaStore().getSource(name);
     if (dataSource == null) {
       throw new KsqlStatementException(String.format(
           "Could not find STREAM/TABLE '%s' in the Metastore",
           name.text()
-      ), statementText);
+      ), statement.getStatementText());
```

Comment:
I looked around to see if `statement` is used anywhere but wasn't able to find it. Did you change everything from `statementText` -> `statement` to allow for extending in the future?